src/javascript/app_2/Stores/Modules/Trading/trade-store.js

```diff
@@ -96,6 +97,7 @@ export default class TradeStore extends BaseStore {
 
     // Last Digit
     @observable last_digit = 5;
+    @observable is_digit_contract = false;
```

Comment:
should this be in the `contract-store`? since it's not directly related to the trade params